tests/VersionRouteTest.php

```diff
@@ -289,4 +290,35 @@ public function testVersion(string $version, bool $isCorrectVersion, string $rou
 
         $response->assertStatus($status);
     }
+
+    public function testWithoutApiVersion(): void
+    {
+        $this->withoutAPIVersion();
+
+        Route::get('/test', function () {
+            return 'test';
+        });
+
+        $response = $this->json('get', '/test');
+
+        $response->assertOk();
+    }
+
+    public function testRouteWithSetApiVersion(): void
+    {
+        $version = $this->createMock(VersionEnumContract::class);
+        $version->value = VersionEnum::v1;
+
+        $this->setAPIVersion($version);
+
+        Route::version($version)->group(function () {
+            Route::get('/test', function () {
+                return 'test';
+            });
+        });
+
+        $response = $this->json('get', '/test/');
+
+        $response->assertOk();
+    }
```

Comment:
need to test `RonasIT\Support\Testing\TestCase` class, for it, let's call the `json` method and assert that it will call the `call` method with the expected arguments